gridformat/vtk/vtp_writer.hpp

```diff
@@ -20,6 +20,8 @@
 #include <gridformat/common/lvalue_reference.hpp>
```

Comment:
Remove include #include <gridformat/common/filtered_range.hpp>

Reply:
Yeah I put that in #322.. but you're right it's cleaner if it happens directly here

Reply:
changed it here